src/zentables/zentables.py

```diff
@@ -12,27 +12,14 @@
 """
 import warnings
 from dataclasses import dataclass
-from typing import (
-    Any,
-    Dict,
-    Iterable,
-    List,
-    Optional,
-    Union,
-)
+from typing import Any, Dict, Generator, Iterable, List, Optional, Union
```

Comment:
Ah, so this `Generator` is not actually the `Generator` you refer to below. You're referencing `np.random.Generator` not `typing.Generator`